src/Router.ts

```diff
@@ -201,13 +250,15 @@ export class Router<C extends Context> extends Evented {
 	private _history?: History;
 	private _routes: Route<Context, Parameters>[];
 	private _started?: boolean;
+	private _outletStack: Map<string, any> = new Map<string, OutletContext>();
```

Comment:
It probably should be named `_outletMap`, started as a stack and then evolved into needing details from the context.